tests/e2e/conftest.py

```diff
@@ -17,6 +17,18 @@ def get_client_instance():
     return client
 
 
+def get_mm_client_instance():
+    load_dotenv()
+    client = C8Client(protocol='https',
+                      host=os.environ.get('FEDERATION_URL'),
+                      port=443,
+                      email=os.environ.get('MM_TENANT_EMAIL'),
+                      apikey=os.environ.get('MM_API_KEY'),
```

Comment:
These two env variables needs to be added to ../../README.md (also maybe you can mention in a comment in the readme file what these two variables mean like Super User (since outside of macrometa no one knows MM tenant)